modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2FileUtil.macro

```diff
@@ -0,0 +1,18 @@
+definition {
+
+	macro assertDownloadedFileIsPresent {
+		var tempDir = PropsUtil.get("output.dir.name");
+
+		var downloadedFilePath = "${tempDir}/${downloadedFileName}";
```

Comment:
@MathausC I was wondering here, when we download something via browser, isn't the file downloaded going to the default path defined in the browser settings? I was just wondering if this directory really works. If you confirmed it, please disregard this comment.